packages/create-svelte/shared/+eslint+prettier+typescript/eslint.config.js

```diff
@@ -0,0 +1,32 @@
+import js from '@eslint/js';
+import ts from 'typescript-eslint';
+import svelte from 'eslint-plugin-svelte';
+import prettier from 'eslint-config-prettier';
+import globals from 'globals';
+
+/** @type {import('eslint').Linter.FlatConfig[]} */
+export default [
+	js.configs.recommended,
+	...ts.configs.recommended,
+	...svelte.configs['flat/recommended'],
+	prettier,
+	{
+		languageOptions: {
+			globals: {
+				...globals.browser,
+				...globals.node
+			}
+		}
+	},
+	{
+		files: ['**/*.svelte'],
+		languageOptions: {
+			parserOptions: {
+				parser: ts.parser
+			}
+		}
+	},
+	{
+		ignores: ['build/', '.svelte-kit/', 'package/', 'vite.config.[jt]s.timestamp-*']
```

Comment:
Not sure what `build` and `package` directories are used for, but they were in the `.eslintignore`.

ESLint doesn't seem to care about `.DS_Store`, `.env*`, `pnpm-lock.yaml`, `package-lock.json` or `yarn.lock`, so I didn't add them.

I've also skipped `node_modules`, since it's ignored by default.

Reply:
I believe `build` is the output directory for adapter-static, etc. and `package` is when building a library with [@sveltejs/package](https://kit.svelte.dev/docs/packaging).

Reply:
> I believe `build` is the output directory for adapter-static, etc.

I think that's correct, found this in docs: https://kit.svelte.dev/docs/adapter-static#options-pages.

> `package` is the directory when packaging the lib directory with @sveltejs/package? (not sure myself)

I've created a new library using `yarn create svelte` and ran `yarn run package`. It created a `dist` directory, so `package` must be for something else.